.htaccess

```diff
@@ -48,7 +48,8 @@ RewriteCond %{REQUEST_FILENAME} !-f
 RewriteRule ^ /engine/dist/index.html [L]
 
 RewriteCond %{REQUEST_URI} ^/storage.*\.(xml|sqlite|db|log|php)$ [OR]
-RewriteCond %{REQUEST_URI} ^/storage/$
+RewriteCond %{REQUEST_URI} ^/storage/$ [OR]
+RewriteCond %{REQUEST_URI} ^/_templates/$
```

Comment:
should user be able to access any files in templates through URL?

Reply:
This restricts only direct access to the dir, same for `storage`.